packages/shared/components/inputs/Pin.svelte

```diff
@@ -5,14 +5,16 @@
     import { Error, Icon } from 'shared/components'
 
     const dispatch = createEventDispatcher()
+    const isWide = () => window.matchMedia('(max-width: 360px)').matches ? true : false
```

Comment:
I don't think we need the ternary here. `window.matchMedia('(max-width: 360px)').matches` returns a boolean value.

Reply:
agree, it's redundant, I'll fix